src/app/pages/IdxPage/index.jsx

```diff
@@ -1,18 +1,33 @@
-import React, { Fragment } from 'react';
+import React, { useContext, Fragment } from 'react';
 import path from 'ramda/src/path';
+import { string } from 'prop-types';
+import { ServiceContext } from '#contexts/ServiceContext';
+import RadioScheduleContainer from '#containers/RadioSchedule';
 import PageContainer from '#lib/pageStyles/PageContainer';
 import FrontPageSection from '#containers/FrontPageSection';
 import idxPageDataPropTypes from '#models/propTypes/idxPage';
 
-const IdxPage = ({ pageData }) => {
+const IdxPage = ({ pageData, radioScheduleEndpointOverride }) => {
```

Comment:
What is the reason why we are not passing  the `radioScheduleEndpointOverride` to the `FrontPage`?

Reply:
This is because `FrontPage` stories do not include a radio schedule https://bbc.github.io/simorgh/?path=/story/pages-front-page--arabic

I added radio schedule in IDX page story so that is why I need an endpoint override

Reply:
Oh yeah, I remember now 👍